services/streammanager/kafka/kafkamanager.go

```diff
@@ -32,6 +37,8 @@ type configuration struct {
 	SaslType      string
 	Username      string
 	Password      string
+	ConvertToAvro bool
+	avroSchema    []schema
```

Comment:
`avroSchemas []avroSchema`
let's use plural on iterables ... or a notation to denote that this is an iterable

Reply:
Implemented.